src/renderer/components/SettingsPage.tsx

```diff
@@ -426,21 +424,36 @@ export default class SettingsPage extends React.PureComponent<Record<string, nev
         }
 
         options.push(
-            <FormCheck>
-                <FormCheck.Input
-                    type='checkbox'
-                    key='inputSpellChecker'
-                    id='inputSpellChecker'
-                    ref={this.useSpellCheckerRef}
-                    checked={this.state.useSpellChecker}
-                    onChange={this.handleChangeUseSpellChecker}
-                />
-                {'Check spelling'}
-                <FormText>
-                    {'Highlight misspelled words in your messages based on your system language configuration. '}
-                    {'Setting takes effect after restarting the app.'}
-                </FormText>
-            </FormCheck>);
+            <>
+                <FormCheck>
+                    <FormCheck.Input
+                        type='checkbox'
+                        key='inputSpellChecker'
+                        id='inputSpellChecker'
+                        ref={this.useSpellCheckerRef}
+                        checked={this.state.useSpellChecker}
+                        onChange={this.handleChangeUseSpellChecker}
+                    />
+                    {'Check spelling'}
+                    <FormText>
+                        {'Highlight misspelled words in your messages based on your system language configuration. '}
```

Comment:
I think this can be expressed better, but it is not the system language anymore
```suggestion
                        {'Highlight misspelled words in your messages based on your language selection. '}
```

Reply:
Agreed on the change, but I think we should maybe 'language preference' instead, since the absence of a selection will default to a system language.